go/carmen/carmen.go

```diff
@@ -277,6 +277,9 @@ type TransactionContext interface {
 	// that gets reset after each transaction.
 	SetTransientState(Address, Key, Value)
 
+	// HasEmptyStorage returns true if account has empty storage.
+	HasEmptyStorage(address Address) bool
```

Comment:
This function should be moved up to the other `State` functions, before the "transient" functions which target another concept.